src/llmq/quorums_blockprocessor.cpp

```diff
@@ -90,7 +90,7 @@ void CQuorumBlockProcessor::ProcessMessage(CNode* pfrom, const std::string& strC
 
         auto members = CLLMQUtils::GetAllQuorumMembers(type, qc.quorumHash);
 
-        if (!qc.Verify(members)) {
+        if (!qc.Verify(members, true)) {
             LOCK(cs_main);
```

Comment:
Doesn't this lock only need to happen for the `Misbehaving`? If that is the case, there isn't really a reason to have it above the print.

Reply:
Yeah, you're right. It's only for `Misbehaving`. I'd like to avoid fixing that for now as I don't want to loose the ACK for this PR if it's not really required...need to finish RC5.